templates/base_layout.html

```diff
@@ -119,62 +135,63 @@
-var t = document.querySelectorAll(target);
-if (t.length >= 1) {
-  if (category.startsWith("nav-")) {
-    t.forEach(function (e) {
-      e.addEventListener("click", function () {
-        sendEvent(category, origin, e.href, e.text);
+  var t = document.querySelectorAll(target);
+  if (t.length >= 1) {
+    if (category.startsWith("nav-")) {
+      t.forEach(function (e) {
+        e.addEventListener("click", function () {
+          sendEvent(category, origin, e.href, e.text);
+        });
       });
-    });
-  } else {
-    // Get form submit events
-    t[0].querySelectorAll("form").forEach(function (e) {
-      var button = e.querySelector(".p-button--positive");
-      if (button) {
-        button.addEventListener("click", function () {
-          sendEvent(
-            "content-cta-0",
-            origin,
-            e.returnURL.value,
-            button.innerText
-          );
+    } else {
+      // Get form submit events
+      t[0].querySelectorAll("form").forEach(function (e) {
+        var button = e.querySelector(".p-button--positive");
+        if (button) {
+          button.addEventListener("click", function () {
+            sendEvent(
+              "content-cta-0",
+              origin,
+              e.returnURL.value,
+              button.innerText
+            );
+          });
+        }
+      });
+      // Get link click events
+      t[0].querySelectorAll("a").forEach(function (e) {
+        if (e.className.includes("p-button--positive")) {
+          category = "content-cta-0";
+        } else if (e.className.includes("p-button")) {
+          category = "content-cta-1";
+        } else {
+          category = "content-link";
+        }
+        e.addEventListener("click", function () {
+          sendEvent(category, origin, e.href, e.text);
         });
-      }
-    });
-    // Get link click events
-    t[0].querySelectorAll("a").forEach(function (e) {
-      if (e.className.includes("p-button--positive")) {
-        category = "content-cta-0";
-      } else if (e.className.includes("p-button")) {
-        category = "content-cta-1";
-      } else {
-        category = "content-link";
-      }
-      e.addEventListener("click", function () {
-        sendEvent(category, origin, e.href, e.text);
       });
-    });
+    }
+  }
   }
-}
-}
 
 function sendEvent(category, origin, destination, label) {
-if (dataLayer) {
-  dataLayer.push({
-    event: "GAEvent",
-    eventCategory: `${site}-${category}`,
-    eventAction: `from:${origin} to:${destination}`,
-    eventLabel: label,
-    eventValue: undefined,
-  });
-}
+  if (dataLayer) {
+    dataLayer.push({
+      event: "GAEvent",
+      eventCategory: `${site}-${category}`,
+      eventAction: `from:${origin} to:${destination}`,
+      eventLabel: label,
+      eventValue: undefined,
+    });
+  }
 }
 
 </script>
-<script src="/js/build/cookie-policy.js"></script>
 <script>
-  cpNs.cookiePolicy();
+  if (cpNs !== undefined) {
```

Comment:
This doesn't work as is, it throws the exception when it tries to run this if statement.
```suggestion
  if (typeof cpNs !== "undefined") {
```